health/health.d/ml.conf

```
@@ -5,21 +5,19 @@
 
 # examples below are commented, you would need to uncomment and adjust as desired to enable them.
 
-# node level anomaly rate example
+# node level anomaly rate
 # https://learn.netdata.cloud/docs/agent/ml#node-anomaly-rate
-# if node level anomaly rate is between 1-5% then warning (pick your own threshold that works best via tial and error).
-# if node level anomaly rate is above 5% then critical (pick your own threshold that works best via tial and error).
-# template: ml_1min_node_ar
-#       on: anomaly_detection.anomaly_rate
-#       os: linux
-#    hosts: *
-#   lookup: average -1m foreach anomaly_rate
-#     calc: $this
-#    units: %
-#    every: 30s
-#     warn: $this > (($status >= $WARNING)  ? (1) : (5))
-#     crit: $this > (($status == $CRITICAL) ? (5) : (100))
-#     info: rolling 1min node level anomaly rate
+# if node level anomaly rate is above 1% then warning (pick your own threshold that works best via trial and error).
+template: ml_1min_node_ar
+      on: anomaly_detection.anomaly_rate
+      os: *
+   hosts: *
+  lookup: average -1m foreach anomaly_rate
+    calc: $this
+   units: %
+   every: 30s
+    warn: $this > 1
+    info: rolling 1min node level anomaly rate
```

```suggestion
    info: rolling 1min node level anomaly rate
    to: silent
```

I think this alert should be silent (no notifications).

---

hmm - I think this would mean that you would probably only see it if you happen to be in the app at the same time of the anomaly and notice a number in the alert tab icon and decide to look at it. 

So you are less likely to actually "discover" it in some sense if no notification is sent, when actually some of the value here could be alerting you about some anomaly that maybe might be something you might have wanted to be made aware of.

I wonder if starting with it as non-silent for a week or so in nighties might make sense and seeing what the feedback is. 

OR we could do opposite - set it to silent to begin with and measure it's occurances in cloud to have a sort of proxy for any potential annoyance factor maybe.

---

We try not to send notifications for anything that doesn't require immediate action/severe enough to wake up you at night. Anomaly is definitely not one of those things. I think it is totally ok to "discover" it/debug any time later.

---

See related PR #15590